tenma/tenmaDcLib.py

```diff
@@ -1048,3 +1051,209 @@ class Tenma72_13330(Tenma72_13320):
+    SERIAL_SETTER_SEPARATOR = ":"
+
+    def getStatus(self):
+        """
+            Returns the power supply status as a dictionary of values
+
+            "ch1Mode ": "C.V" or "C.C",
+            "output ": "ON" or "OFF",
+            "V/C priority ": "Current priority" or "Voltage priority",
+            "beep ": "ON" or "OFF",
+            "lock ": "ON" or "OFF",
+
+            :return: Dictionary of status values
+        """
+        self._sendCommand("STATUS?")
+        statusBytes = self._readBytes()
+
+        # 72-13360 sends two bytes back, the second being '\n'
+        status = statusBytes[0]
+
+        ch1mode = (status & 0b00000001)
+        output = (status & 0b00000010)
+        current_priority = (status & 0b00000100)
+        beep = (status & 0b00010000)
+        lock = (status & 0b00100000)
+
+        return {
+            "ch1Mode ": "C.V" if ch1mode else "C.C",
+            "output ": "ON" if output else "OFF",
+            "V/C priority ": "Current priority" if current_priority else "Voltage priority",
+            "beep ": "ON" if beep else "OFF",
+            "lock ": "ON" if lock else "OFF",
+        }
+    
+    def readCurrent(self, channel=""):
+        """
+            Reads the current setting of the PSU
+
+            :param channel: Is ignored but there for backwards compatability
+            :return: Current for the PSU in Amps as a float
+        """
+        return super().readCurrent("")
+
+    def setCurrent(self, mA):
+        """
+            Sets the current of the PSU
+
+            :param mA: Current to set, in mA
+            :raises TenmaException: If the current does not match what was set
+            :return: The current the PSU was set to in Amps as a float
+        """
+        return super().setCurrent("", mA)
+
+    def readVoltage(self, channel=""):
+        """
+            Reads the voltage setting of the PSU
+
+            :param channel: Is ignored but there for backwards compatability
+            :return: Voltage in Volts as a float
+        """
+        return super().readVoltage("")
+
+    def setVoltage(self, mV):
+        """
+            Sets the voltage of the PSU
+
+            :param mV: voltage to set, in mv
+            :raises TenmaException: If the voltage does not match what was set
+            :return: The voltage the PSU is set to in Volts as a float
+        """
+        return super().setVoltage("", mV)
+
+    def runningCurrent(self):
+        """
+            Returns the current read
+
+            :return: The running current in Amps as a float
+        """
+        return super().runningCurrent("")
+
+    def runningVoltage(self):
+        """
+            Returns the voltage read
+
+            :return: The running voltage in volts as a float
+        """
+        return super().runningVoltage("")
+
+    def setTracking(self, trackingMode):
+        raise NotImplementedError("72-13360 does not support tracking")
+    
+    def setVoltagePriority(self):
```

Comment:
Ideally, up until now we forced all the functions to be available in `Tenma72Base` so we can freely call it in all PSUs but raise a `NotImplemented` error